### PR TITLE
XIVY-14657 Statistics page

### PR DESCRIPTION
**Pull request by jackra1n**
I feel like theres something missing and I'm not sure what kinds of statistics to show. Are some of the current ones not worth showing and show something else instead? For now I looked a bit at what statistics portal is showing and some of my thoughts. Please leave feedback 😃 

image
image

### CODE REVIEW

File: dev-workflow-ui/src/ch/ivyteam/workflowui/StatisticsBean.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+@ManagedBean
+@ViewScoped
+public class StatisticsBean {
+
+  public long getDueToday() {
+    var aggregates = "expiryTimestamp:bucket:day";
+    var filter = "canWorkOn,expiryTimestamp:>=now/d <now/d+d";
+    var aggregation = WorkflowStats.current().task().aggregate(aggregates, filter);
+    Ivy.log().info("Due today: " + aggregation);
```
Comment (ivy-lli) [inline dev-workflow-ui/src/ch/ivyteam/workflowui/StatisticsBean.java:28]: remove

File: dev-workflow-ui/webContent/view/statistics.xhtml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<h:body>
+  <ui:composition template="../includes/template/template.xhtml">
+    <ui:define name="content">
+      <f:event type="preRenderView" listener="#{loginIvyDevWfBean.redirectIfNotAdmin}" />
+      <div class="card">
+        <div class="flex align-items-center mb-3">
+          <i class="pr-3 si si-network-browser card-title-icon" />
+          <h2 class="m-0">Statistics</h2>
+        </div>
+      </div>
```
Comment (ivy-lli) [inline dev-workflow-ui/webContent/view/statistics.xhtml:14]: IMO this looks not good, to have the title separated in its own card. If not better possible I would remove it completely.

**Review by ivy-lli — COMMENTED** [review]
Problems:
- If no task exists the `Tasks by state` is empty.
- the border of the donat looks a little bit ugly in dark mode.

I'm not sure about the value of some statistics here, but honestly I also not 100% what is possible by the API.
- I think the `tasks by state` on the home is good.
- Not sure about the value of `my done tasks` on home.
- `Due today` completely senseless, as no one should have the dev-wf-ui open for multiple days IMO
- `Tasks open` already integrated in the `tasks by state`
- `all tasks` also kind of already there in the `tasks by state`
- `tasks by priority` not sure of the value of this, as I'm not sure how many use this priority stuff, but ok.

General I imagined a little bit more technical statistics here. As I mentioned before, I'm not sure what the statistics API is capable of but something like:
- how many signals fired
- or maybe some time stuff, timeline when new tasks where created
- or more cases stuff

**Comment by ivy-cst** [comment]
@alexsuter Any good ideas what else we could show?

**Comment by alexsuter** [comment]
> @alexsuter Any good ideas what else we could show?

Not yet. But you may have a look into portal

**Comment by jackra1n** [comment]
We decided with @ivy-cst to not put any statistics on home page at least for now
statistics4
statistics-nodata

**Review by ivy-lli — COMMENTED** [review]
Looks good, some final points:
- Why do you not use the full width?
- I would set 0 as the minimal point for the empty graphs

**Comment by jackra1n** [comment]
Unfortunately I did not find a good way to make the line chart not show y-axis values below 0. But this is also only the case when there is no data for the graph.
image
image

**Pull request merged**